projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -78,15 +78,15 @@ export class ListViewComponent implements OnChanges {
         this.view = view;
     }
 
-    emitSelectedResource(id: string) {
+    emitSelectedResources(resInfo: object) {
```

Comment:
It would be better to have correct type instead of object.

Reply:
Maybe you can create an `Interface` with this object. It will help in the implementation in DSP-APP: